### PR TITLE
Fix: issue while scroll search list

### LINKED ISSUE #183: Search scrolling on mobile busted
https://github.com/ApollosProject/apollos-embeds/assets/2659478/bb77849b-bcc7-4a5c-bb6f-8d832581da16

### PR DESCRIPTION
**Pull request by rajitharussel**
## 🐛 Issue

Closes #183 
the search results container does not update when input focused but keyboard hidden. 

#183 

## ✏️ Solution

updated container height to `100vh` so issue will not happen

## 🔬 To Test

1. open web-embed application and focus on search bar. and then scroll up starting from the bottom while keyboard is hidden. 
2.

## 📸 Screenshots

https://github.com/ApollosProject/apollos-embeds/assets/138706838/ad226b39-75b0-462f-90aa-35c3c5c01a90

https://github.com/ApollosProject/apollos-embeds/assets/138706838/c76e6064-40eb-4391-a6c6-a53f82782287

Screenshot 2024-03-07 at 19 39 59

### CODE REVIEW

**Comment by rajitharussel** [comment]
> @rajitharussel can you include a picture of the search box opening on a desktop sized window? Thank you!

updated

**Comment by redreceipt** [comment]
@rajitharussel this has been sitting for three weeks, please make sure there is a requestor selected and PRs are moving along. If you don't know who to add, add me.

**Comment by rajitharussel** [comment]
> @rajitharussel this has been sitting for three weeks, please make sure there is a requestor selected and PRs are moving along. If you don't know who to add, add me.

got it.

File: web-embeds/src/index.css [hunk 1]
```diff
@@ -38,3 +38,9 @@
 .apollos-widget input:focus-visible {
   outline: none;
 }
+
+@media only screen and (min-device-width: 320px) {
+  #apollos-project-widget > div > div > div > div {
```
Comment (redreceipt) [inline web-embeds/src/index.css:43]: If the DOM tree changes, will this still work?

Comment (rajitharussel) [reply web-embeds/src/index.css]: Moved to more secure place

**Comment by redreceipt** [comment]
@vinnyjth @conrad-vanl could one of you verify this? Seems simple enough but my CSS is no good.

File: packages/web-shared/components/Modal/Modal.styles.js [hunk 2]
```diff
@@ -26,6 +26,9 @@ const slideOut = keyframes`
 const Modal = withTheme(styled.div`
   width: 100%;
   height: 100%;
+  @media screen and (max-width: ${themeGet('breakpoints.sm')}) {
```
Comment (conrad-vanl) [inline packages/web-shared/components/Modal/Modal.styles.js:29]: very nice! much better then in .css

**Review by conrad-vanl — APPROVED** [review]

**Pull request merged**